SwiftKit.podspec

```diff
@@ -8,52 +8,63 @@
-  spec.author           = { "Tadeas Kriz" => "tadeas@brightify.org", "Filip Dolnik" => "filip@brightify.org" }
-  spec.source           = { 
-    :git => "https://github.com/brightify/SwiftKit.git", 
-    :tag => spec.version.to_s
-  }
-  spec.social_media_url = 'https://twitter.com/BrightifyOrg'
-
-  spec.platform     = :ios, '8.0'
-  spec.requires_arc = true
-
-  spec.subspec 'Events' do |events|
-    events.source_files = 'Events/**/*.swift'
-    events.frameworks = 'UIKit'
-  end
-
-  spec.subspec 'Preferences' do |preferences|
-    preferences.source_files = 'Preferences/**/*.swift'
-    preferences.frameworks = 'Foundation'
-    preferences.dependency 'SwiftKit/Events'
-  end
-
-  spec.subspec 'Injection' do |injection|
-    injection.source_files = 'Injection/**/*.swift'
-    injection.frameworks = 'Foundation'
-  end
-
-  spec.subspec 'Router' do |router|
-    router.source_files = 'Router/**/*.swift'
-    router.frameworks = 'Foundation'
-    router.dependency 'Alamofire'
-    router.dependency 'ObjectMapper'
-    router.dependency 'SwiftyJSON'
-  end
-
-  spec.subspec 'ViewComposer' do |composer|
-    composer.source_files = 'ViewComposer/**/*.swift'
-    composer.frameworks = 'UIKit'
-    composer.dependency 'SwiftKit/Events'
-  end
-
-end
+    spec.homepage         = "https://github.com/brightify/SwiftKit"
+    spec.license          = 'MIT'
+    spec.author           = { "Tadeas Kriz" => "tadeas@brightify.org", "Filip Dolnik" => "filip@brightify.org" }
+    spec.source           = {
+        :git => "https://github.com/brightify/SwiftKit.git",
+        :tag => spec.version.to_s
+    }
+    spec.social_media_url = 'https://twitter.com/BrightifyOrg'
+
+    spec.platform     = :ios, '8.0'
+    spec.requires_arc = true
+
+    spec.subspec 'Events' do |events|
+        events.source_files = 'Events/**/*.swift'
+        events.frameworks = 'UIKit'
+    end
+
+    spec.subspec 'Preferences' do |preferences|
+        preferences.source_files = 'Preferences/**/*.swift'
+        preferences.frameworks = 'Foundation'
+        preferences.dependency 'SwiftKit/Events'
+    end
+
+    spec.subspec 'Injection' do |injection|
+        injection.source_files = 'Injection/**/*.swift'
+        injection.frameworks = 'Foundation'
+    end
+
+    spec.subspec 'Router' do |router|
+        router.source_files = 'Router/**/*.swift'
+        router.frameworks = 'Foundation'
+        router.dependency 'Alamofire'
+        router.dependency 'ObjectMapper'
+        router.dependency 'SwiftyJSON'
+    end
+
+    spec.subspec 'ViewComposer' do |composer|
+        composer.source_files = 'ViewComposer/**/*.swift'
+        composer.frameworks = 'UIKit'
+        composer.dependency 'SwiftKit/Events'
+    end
+
+    spec.subspec 'ViewComposer' do |composer|
+        composer.source_files = 'ViewComposer/**/*.swift'
+        composer.frameworks = 'UIKit'
+        composer.dependency 'SwiftKit/Events'
+    end
+
+    spec.subspec 'TestUtils' do |composer|
```

Comment:
As it is now, each project would get TestUtils as a dependency, which makes no sense.